src/content/community/releases/_index.en.md

```diff
@@ -42,6 +42,7 @@ weight = 40
   liveness probes.
 * Submariner now uses case-insensitive comparison while parsing CNI names.
 * Enhanced Calico CNI detection now includes searching for calico-node CNI pods when the calico-config map is not detected.
+* Submariner now automatically creates Calico IPPools for the subnets associated with the Pod and Service CIDRs of remote clusters.
```

Comment:
Maybe we should mention here that running [Calico API server](https://docs.tigera.io/calico/latest/operations/install-apiserver#how-to) is a prerequisite for this capability